azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -299,6 +307,35 @@ public void uploadLogFile(final int execId, final String name, final int attempt
     this.executionLogsDao.uploadLogFile(execId, name, attempt, files);
   }
 
+  @Override
+  public void appendLogs(final int execId, final String name, final String logs) throws
```

Comment:
This is introducing a path where the webserver will update the flow-container logs stored in the db (possibly after the container is already gone).  This is not a problem per se, but does introduce a pattern of log updates we didn't have before. 
Would be good to keep the rest of the folks in the team in the loop for this as well as deploy and monitor the first fews clusters with this change more cautiously.